libraries/engage/core/router/helpers.js

```diff
@@ -0,0 +1,37 @@
+import { UIEvents } from '@shopgate/pwa-core';
+
+export const NAVIGATION_PUSH = 'navigation.push';
+export const NAVIGATION_POP = 'navigation.pop';
+export const NAVIGATION_REPLACE = 'navigation.replace';
+export const NAVIGATION_RESET = 'navigation.reset';
+
+/**
+ * Performs the PUSH navigation action.
+ * @param {Object} params The history params.
+ */
+export function push(params) {
```

Comment:
What about also providing a way to update a route. I know from the PS team that they already needed that in an extension. By now it can only be done importing the conductor, and invoking its `update` method.
The major problem here is the routeId which needs to be passed with the parameters for `update`. It should be the route of the calling component by default, but it might also be necessary to use the id of another route, like we do it with the filters.

Reply:
I provided the helper in question.